style.css

```diff
@@ -85,3 +89,191 @@ img {
   filter: grayscale(100%);
   gap: 20px;
 }
+
+#social_icons {
+  padding-left: 8px;
+}
+
+.social_media h3 {
+  margin: 0;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: auto;
+  row-gap: 100px;
+  margin-top: 42px;
+  margin-bottom: 114px;
+}
+
+.card {
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  min-height: 90%;
+  padding: 16px;
+  margin: 0 24px;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+}
+
+.project_name {
+  color: #172b4d;
+  margin-top: 12px;
+}
+
+#title {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.project_info {
+  display: flex;
+  gap: 5px;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.ul {
+  display: flex;
+}
+
+.company p {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  text-transform: uppercase;
+}
+
+.dot {
+  width: 8px;
+  height: 8px;
+}
+
+.role {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  margin-right: 30px;
+}
+
+.year {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.description {
+  width: 90%;
+  margin-top: 10px;
+}
+
+.description p {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.languages {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  margin-top: 12px;
+  list-style: none;
+  padding: 0;
+}
+
+#lang {
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.button-container {
+  margin-top: 12px;
+  height: 48px;
+}
+
+.btn {
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  height: 48px;
+  outline: none;
+  background-color: white;
+  color: #396df2;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  cursor: pointer;
+  padding: 12px;
+}
+
+.btn:hover {
+  background: #6070ff;
+  color: #fff;
+  transition: all 0.5s ease-out;
+}
+
+.about {
+  min-height: 100vh;
+  background-color: #fff;
+  border-top-right-radius: 100px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  padding: 12px 24px 0 24px;
+}
+
+.skills {
```

Comment:
- [ ] Good job so far But I noticed that your design is a little bit different from the Figma template. Please make sure that your technology title has the same `font size` as defined on the Figma template. Also, don't forget to have the same arrow icons as defined in the Figma design. :+1: 

- [ ] I also noticed that your design is different from the Figma template. Please make sure that you stick to the Figma design as much as possible. As a developer it is very important to pay attention to the details :+1:  
Please check the screenshots below to see the differences between your design and Figma :arrow_down: 

#### Your design
![Screenshot from 2022-10-28 15-41-24(1)](https://user-images.githubusercontent.com/6615394/198680111-974f4b15-3240-4f72-950f-de813c6842d2.jpg)

#### Figma design
![Screenshot from 2022-10-28 15-52-32](https://user-images.githubusercontent.com/6615394/198680419-3116ccc2-b676-467b-a60e-880e78fad732.png)